docs/release-notes.md

```diff
@@ -19,58 +19,51 @@
 
 ## What's New?
 
-:material-star:** Database update required **
-An upgrade to the `rpsdb` database is required.
+:material-new-box:** New Feature: 802.1X Wireless Configuration Support **
 
-[See Upgrade Toolkit Version docs for SQL scripts to add the new table and columns](../Deployment/upgradeVersion/).
+For devices deployed in a secure enterprise network, configuring AMT to authenticate with the network is important so that connection can be maintained with the device even when the OS is not available. With this release, we now support configuring AMT with certificate-based 802.1x configurations (EAP-TLS, PEAPv1/EAP-GTC, EAP-FAST/GTC, EAP-FAST/TLS) for wireless networks.  [Find more information about 802.1x configuration here](../References/EA/ieee8021xconfig/). 
```

Comment:
Bad link, /Reference/ not /References/.  Also maybe we jump them right to Wireless:
```
../Reference/EA/ieee8021xconfig/#wireless-8021x-configuration
```

Also need to update the supported configs (only EAP-TLS and PEAPv0/EAP-MSCHAPv2)

Reply:
I didn't think we support MSCHAPv2 yet.  that will be later

Reply:
Yeah was just checking myself.  I'll open PR to update 8021x docs then to remove.